metadata-ingestion/scripts/install_deps.sh

```diff
@@ -19,7 +19,13 @@ else
             xz-devel \
             libxml2-devel \
             libxslt-devel \
-	    krb5-devel
+            krb5-devel
+    elif command -v apk; then
```

Comment:
This works but I think it would be better to check for distribution vs apk/yum command as there are other distros which has `yum` but the package names are different. -> https://www.cyberciti.biz/faq/find-linux-distribution-name-version-number/

This is more an fyi

Reply:
I think for `apk` it's fine since it's the alpine package keeper. Didn't know that yum could have issues - it hasn't been an issue thus far so I think I'll leave it as-is, but good to be aware of that